src/main/java/gov/cancer/pageobject/cts/advanced_search_page_components/CancerTypeSection.java

```diff
@@ -101,4 +105,20 @@ public AutoSuggestField getCancerTypeCondition() {
   public Button getAllButton() {
     return all;
   }
+
+  /**
+   * Getter for  Cancer type input box
+   * @return
+   */
+  public WebElement getCancerTypeInputBox() {
```

Comment:
all WebElement accesses should be private or wrapped in the Object

Reply:
fixed this